setup.py

```diff
@@ -57,6 +57,7 @@
         'mysql': ['sqlalchemy', 'pymysql'],
         'oracle': ['sqlalchemy', 'cx_Oracle'],
         'postgresql': ['sqlalchemy', 'psycopg2-binary'],
+        'postgresql-pg8000': ['sqlalchemy', 'pg8000'],
```

Comment:
```suggestion```

Users can install different drivers independently.